packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.types.ts

```diff
@@ -209,6 +209,12 @@ export interface IComboBoxStyles {
    * Styles for a divider in the options.
    */
   divider: IStyle;
+
+  /**
+  * Custom width for dropdown. If value is 0, width of the input field is used.
+  * @default 0
+  */
+  dropdownWidth?: number;

```

Comment:
This prop should probably be moved to IComboBoxProps. The styles interface is for IStyle type props (IStyle group css property/value pairs).  @dzearing  agree?

Reply:
Makes sense. I moved the prop into combobox.props. Dropdown did the same way